docs/resources/core/instance.md

```diff
@@ -35,9 +52,11 @@ The following arguments are supported:
 * `create_vnic_details` - (Optional) Details for creating a new VNIC. See [Create Vnic Details](https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/requests/CreateVnicDetails).
 * `shape` - (Required) The shape of an instance.
 * `subnet_id` - (Optional) The OCID of the subnet. This must be specified either here or in `create_vnic_details`.
+* `hostname_label` - (Optional) The hostname for the VNIC's primary private IP. This may be specified either here or in `create_vnic_details`.
```

Comment:
I think we want to indicate that this is deprecated.  Does our code prefer the value of create_vnic_details.hostname_label over this?